pom.xml

```diff
@@ -142,13 +142,12 @@
     <profile>
       <id>spring-cloud-azure-5.x</id>
       <modules>
-<!--will be open after spring-data-cosmos supported-->
-<!--        <module>-->
-<!--          appconfiguration/spring-cloud-azure-appconfiguration-config/spring-cloud-azure-appconfiguration-config-convert-sample/spring-cloud-azure-appconfiguration-config-convert-sample-complete-->
-<!--        </module>-->
-<!--        <module>-->
-<!--          appconfiguration/spring-cloud-azure-appconfiguration-config/spring-cloud-azure-appconfiguration-config-convert-sample/spring-cloud-azure-appconfiguration-config-convert-sample-initial-->
-<!--        </module>-->
+        <module>
+          appconfiguration/spring-cloud-azure-appconfiguration-config/spring-cloud-azure-appconfiguration-config-convert-sample/spring-cloud-azure-appconfiguration-config-convert-sample-complete
+        </module>
+        <module>
+          appconfiguration/spring-cloud-azure-appconfiguration-config/spring-cloud-azure-appconfiguration-config-convert-sample/spring-cloud-azure-appconfiguration-config-convert-sample-initial
```

Comment:
The appconfiguration samples should work well without any changes under Spring Boot 2 and 3, if not, a conversion doc should be added to the sample project folder, see more from https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main#samples-for-spring-boot-3